interfaces/Portalicious/package.json

```diff
@@ -34,8 +34,8 @@
     "@angular/router": "^18.2.11",
     "@microsoft/applicationinsights-web": "^3.3.4",
     "@tanstack/angular-query-experimental": "^5.60.0",
-    "@azure/msal-angular": "^3.1.0",
-    "@azure/msal-browser": "^3.27.0",
+    "@azure/msal-angular": "3.0.25",
```

Comment:
If we need to block these versions, we should also enforce it via the dependabot file

Reply:
Good one, let me figure that one out

Reply:
We _should_ keep an eye on the _next, fixed_ release of both packages though...

So I think we can remove them from the Dependabot-ignore list fairly soon... (especially now that we let it check less often)

Reply:
I've subscribed to the "Releases" of the repo; But don't know yet how often these are created... and if I would notice the GitHub-notification (or maybe it'll be an email)